src/main/java/net/minecraftforge/common/util/Constants.java

```diff
@@ -49,4 +49,132 @@
+         *        N
+         *    0 | 1 | 2
+         *    ----------
+         * W  3 | 4 | 5  E
+         *    ----------
+         *    6 | 7 | 8
+         *        S
+         * Setting the data to 4 will mean the particles won't spawn
+         */
+        public static final int DISPENSER_SMOKE                 = 2000;
+
+        /**
+         * data is the state id of the block broken {@link net.minecraft.block.Block#getStateId}
+         */
+        public static final int BREAK_BLOCK_EFFECTS             = 2001;
+        /**
+         * data is the rgb color int that should be used for the potion particles
+         */
+        public static final int SPLASH_POTION_EFFECT            = 2002;
+        public static final int ENDER_EYE_SHATTER               = 2003;
+        public static final int MOB_SPAWNER_PARTICLES           = 2004;
+        /**
+         * data is the amount of particles to spawn. If data is 0 then there will be 15 particles spawned
+         */
+        public static final int BONEMEAL_PARTICLES              = 2005;
+        public static final int DRAGON_FIREBALL_HIT             = 2006;
+        /**
+         * data is the rgb color int that should be used for the potion particles
+         */
+        public static final int LINGERING_POTION_EFFECT         = 2007;
+        public static final int GATEWAY_SPAWN_EFFECTS           = 3000;
+        public static final int ENDERMAN_GROWL_SOUND            = 3001;
+    }
+
+
+    /**
+     * The flags used when calling {@link net.minecraft.world.World#setBlockState}
+     * Can be found from World#markAndNotifyBlock and RenderGlobal#notifyBlockUpdate
+     * Flags can be OR-ed
+     */
+    public static class BlockFlags {
+        /**
+         * Calls neighborChanged on surrounding blocks
+         */
+        public static final int NOTIFY_NEIGHBORS     = 0b00001;
+        /**
+         * Sends the update to the client
+         */
+        public static final int SEND_TO_CLIENTS      = 0b00010;
+        /**
+         * Stops the blocks from being marked for a render update
+         */
+        public static final int NO_RERENDER          = 0b00100;
+        /**
+         * Makes the block be re-rendered immediately, on the main thread
+         */
+        public static final int RERENDER_MAIN_THREAD = 0b01000;
+        /**
+         * Disables observers from seeing this update
+         */
+        public static final int NO_OBSERVERS         = 0b10000;
+
+        public static final int DEFAULT = NOTIFY_NEIGHBORS | SEND_TO_CLIENTS;
+        public static final int DEFAULT_AND_RERENDER = DEFAULT | RERENDER_MAIN_THREAD;
+    }
+
+    /**
+     * The mutex bits used for AI tasks {@link net.minecraft.entity.ai.EntityAIBase#setMutexBits(int)}
+     * Bits can be OR-ed
+     */
+    public static class AiMutexBits {
+        public static final int MOTION  = 0b001;
+        public static final int LOOKING = 0b010;
+        public static final int JUMPING    = 0b100;
```

Comment:
Weird extra spacing? Also agree with liach that the tense should stay consistent. `MOVE`, `LOOK`, `JUMP` for instance.